scripts/histmakers/mz_lowPU.py

```diff
@@ -94,9 +94,16 @@ def build_graph(df, dataset):
 
     if dataset.name in sigProcs:
 
-        axes_xnorm = [axis_xnorm, common.axis_recoil_gen_ptZ_lowpu, axis_charge]
-        cols_xnorm = ["xnorm", "ptVgen", "TrigMuon_charge"]
-        
+        axes_xnorm = [axis_xnorm, common.axis_recoil_gen_ptZ_lowpu, axis_charge, differential.axis_fiducial]
+        cols_xnorm = ["xnorm", "ptVgen", "TrigMuon_charge", "fiducial"]
+
+        df = theory_tools.define_prefsr_vars(df)
+        df = df.Define("fiducial", """ (abs(genl.eta()) < 2.4) && (abs(genlanti.eta()) < 2.4) 
+            && (genl.pt() > 26) && (genlanti.pt() > 26) 
+            && (genl.pt() < 55) && (genlanti.pt() < 55) 
+            && (massVgen > 60) & (massVgen < 120) 
+            && ptVgen < 150 """)
+
```

Comment:
Should we put these constant in some configuration script? It might be very easy to use inconsistent values elsewhere otherwise